LISISO/install.sh

```diff
@@ -70,6 +70,20 @@ if [ $distro_version = "unknown" ]; then
     exit 1
 fi
 
+OracleDistroName=$(grep -ihs "Orac" /etc/oracle-release)
+OracleDistroVersion=[52,53,54,55,56,57,58,59,510,511,60,61,62,63]
+case $OracleDistroName in
```

Comment:
Please perform unit testing for Oracle distro?
Also check if it can be tested for this negative condition

Reply:
Tested for Oracle distro for negative condition.

Reply:
[root@localhost LISISO]#./install.sh 
raji 58
installation is not supported for Oracle distros from series 52 to 63
[root@localhost LISISO]# ./upgrade.sh 
raji 59
Upgrade is not supoorted for Oracle distros frpm 52 to 63